pkg/database/postgres/tables.go

```diff
@@ -213,7 +213,7 @@ order by c.ordinal_position`
 }
 
 func (p *PostgresConnection) GetTableSchema(tableName string) ([]*types.Column, error) {
-	query := "select column_name, data_type, character_maximum_length, column_default, is_nullable from information_schema.columns where table_name = $1 and table_schema = $2"
+	query := "select column_name, data_type, character_maximum_length, column_default, is_nullable from information_schema.columns where table_name = $1 and catalog = $2"
```

Comment:
It's `table_catalog`

Reply:
sorry, i worked completely mechanical here because i forgot about this pr and we don't really need it anymore

Reply:
should be good now :)